msg/mavlink_log.msg

```diff
@@ -1,6 +1,6 @@
 uint64 timestamp		# time since system start (microseconds)
 
-char[50] text
+char[127] text
```

Comment:
Any reason why you picked this number? If so please add a comment or explain it in the commit message.

Reply:
In the PR I mentioned "which is the same size as in msg/log_message". So you can use functions like `mavlink_and_console_log_info` and expect the same message in mavlink and in the px4 log.

Reply:
Ah, that makes sene.